### ISSUE TITLE
Set up domain name

### ISSUE THREAD

**Issue by emhoracek** [positiondev/mxlinche#19]
So, a tricky thing about the SSL certificate -- I put grafnyc-staging.positiondevapp.com on the cert, but not grafnyc.org.

So I'm going to backup the database, add grafnyc.org to the cert, and restore the database.

**Comment by emhoracek** [comment]
@alvaroov It looks like the new domain isn't going to work with the current TLS certificate, but I also can't edit the certificate to add new domains. SO I'm going to move forward on #17 and set up a "prod" server that grafnyc.org will point to.

**Comment by emhoracek** [comment]
Started fresh today. One of the problems I was having last week was that every time I ran into a problem with deploying the new server, I tried to start from scratch with a new server, which had a new IP address, which I had to change in Cloudflare to make the domain point to the right server. Then, those changes had to propagate through the whole Internet. Now, I'm going to use an "Elastic IP", which will allow us to change servers without changing the DNS. 

I've provisioned the Elastic IP, but maybe due to the problems from last week, it may not be linked to the domain name for up to 25 hours. (`nslookup -debug grafnyc.org` gives a TTL of 91969)
